dialog/custom.go

```diff
@@ -55,6 +55,14 @@ func (d *CustomDialog) SetButtons(buttons []fyne.CanvasObject) {
 	d.setButtons(container.NewGridWithRows(1, buttons...))
 }
 
+// SetIcon sets an icon to be shown on the right of the dialog, like the NewInformation dialog.
```

Comment:
I think we should mention top right and ignore the example.
```suggestion
// SetIcon sets an icon to be shown in the top right of the dialog.
```